### ISSUE TITLE
[Bug][Azure DevOps] Entities Column Missing

### ISSUE THREAD

**Issue by awickham10** [apache/devlake#5697]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

When standing up a new v0.18.0-beta1 deployment and trying to create scope config, it errors indicating that the entities column does not exist on the _tool_azuredevops_gitrepositoryconfigs table.

```
time="2023-07-18 16:55:18" level=info msg="/app/impls/dalgorm/dalgorm.go:196 Error 1054 (42S22): Unknown column 'entities' in 'field list'
[1.704ms] [rows:0] INSERT INTO `_tool_azuredevops_gitrepositoryconfigs` (`created_at`,`updated_at`,`entities`,`connection_id`,`name`,`refdiff`,`deployment_pattern`,`production_pattern`) VALUES ('2023-07-18 16:55:18.926','2023-07-18 16:55:18.926','["CODE","CODEREVIEW","CROSS","CICD"]',1,'DevSecOps Scope Config',NULL,'(deploy|push-image)','')"
```

### What do you expect to happen

The column to exist. 

### How to reproduce

Stand up a new v0.18.0-beta1 environment. The 

### Version

v0.18.0-beta1

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by keon94** [comment]
I see we didn't add a migration-script for it, that's why. Should be a simple code fix.

**Comment by keon94** [comment]
I have the column on my db:
![image](https://github.com/apache/incubator-devlake/assets/25063936/c07d19ca-bc30-4b74-abdc-97f6cf676c50)

Could you check if you have migration (script-version: 20230606165630) in the devlake_migration_history table?

**Comment by awickham10** [comment]
Looks like it was run.

```
+-------------------------+----------------+--------------------------------------+-------------+
| created_at              | script_version | script_name                          | comment     |
+-------------------------+----------------+--------------------------------------+-------------+
| 2023-07-18 16:05:49.588 | 20230606165630 | rename_tx_rule_table_to_scope_config | azuredevops |
+-------------------------+----------------+--------------------------------------+-------------+
```

**Comment by keon94** [comment]
@awickham10 Can you check in the db if the gitrepositoryconfigs table is indeed missing that column? Is there any error in the startup logs of the server?

**Comment by keon94** [comment]
Ok, I figured it out, and this issue only gets triggered if the entities column already existed on the table. #5714 fixes this, and will be cherry-picked into the next v0.18 beta.

**Comment by Startrekzky** [comment]
Hi @awickham10 , this fix is included in the v0.18.0-beta3.
